### PR TITLE
Updated the readme (example of Effect decorator on a function)

### PR DESCRIPTION
**Pull request by dsebastien**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,30 @@ export class SomeCmp {
...
+In the `AuthEffects` example above, the `@Effect` decorator was placed on a member property of the class.
+That will work fine if you declare the stateUpdate member property in the constructor but will fail at runtime if you declare those outside of the constructor due to the order in which TypeScript emits the JS code. For more details, see #17.
+
+If you declare member properties outside of the constructor, you MUST add the `@Effect` decorator to a function as in the example below:
+
+```
+class AuthEffects {
+   private update$:StateUpdates<any>;
+
+   @Effect
```
Comment (awerlang) [inline README.md]: Should be `@Effect()`

File: README.md [hunk 2]
```diff
@@ -112,6 +112,30 @@ export class SomeCmp {
...
 ```
 
+### Declaring effects as functions
+In the `AuthEffects` example above, the `@Effect` decorator was placed on a member property of the class.
+That will work fine if you declare the stateUpdate member property in the constructor but will fail at runtime if you declare those outside of the constructor due to the order in which TypeScript emits the JS code. For more details, see #17.
+
+If you declare member properties outside of the constructor, you MUST add the `@Effect` decorator to a function as in the example below:
+
+```
+class AuthEffects {
```
Comment (awerlang) [inline README.md:124]: Should have @Injectable() prepended

**Comment by dsebastien** [comment]
Thanks, I've updated the PR

**Comment by MikeRyanDev** [comment]
Sorry for taking so long to get to this, but an example of decorating a method with `@Effect()` is included in the docs.

Thanks for the contribution!

**Pull request closed without merging**